wisdem/floatingse/member.py

```diff
@@ -17,11 +17,13 @@
 
 
 class CrossSection(object):
-    def __init__(self, D=0.0, t=0.0, A=0.0, Asx=0.0, Asy=0.0, Ixx=0.0, Iyy=0.0, Izz=0.0, E=0.0, G=0.0, rho=0.0):
+    def __init__(
+        self, D=0.0, t=0.0, A=0.0, Asx=0.0, Asy=0.0, Ixx=0.0, Iyy=0.0, Izz=0.0, E=0.0, G=0.0, rho=0.0, sigy=0.0
+    ):
         self.D, self.t = D, t  # Needed for OpenFAST
         self.A, self.Asx, self.Asy = A, Asx, Asy
         self.Ixx, self.Iyy, self.Izz = Ixx, Iyy, Izz
-        self.E, self.G, self.rho = E, G, rho
+        self.E, self.G, self.rho, self.sigy = E, G, rho, sigy
```

Comment:
is self.sigy = E right?

Reply:
I think it is okay.  It is a group assignment in python. `a, b = 2, 3` sets `a=2` and `b=3`.

Reply:
oooh sure I had the alignment off and I got confused